test/e2e/broker_default_test.go

```diff
@@ -92,7 +92,7 @@ func TestDefaultBrokerWithManyTriggers(t *testing.T) {
 			},
 			deprecatedTriggerFilter: true,
 		}, {
-			name: "test default broker with many attribute triggers",
+			name: "test default broker with many attrs triggers",
```

Comment:
I wonder why you're switching to using more obscure abbreviations instead of full words in the strings...